account_global_discount/__manifest__.py

```diff
@@ -9,6 +9,9 @@
     "website": "https://github.com/OCA/account-invoicing",
     "license": "AGPL-3",
     "depends": ["account", "base_global_discount"],
+    "excludes": [
```

Comment:
As I will use nobump, in this manifest the version has to be bumped manually.

Reply:
Nice catch !